### PR TITLE
Minor edits to docs

### PR DESCRIPTION
**Pull request by brannondorsey**

### CODE REVIEW

**Comment by brannondorsey** [comment]
Ready to merge this guy once it's been reviewed.

File: docs/source/exceptions.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Exceptions
 
-The Runway Model SDK defines several custom exception types. All of these exceptions derive from the base ``RunwayError`` class, which itself derives from the standard Python ``Exception`` class. Each ``RunwayException`` contain error `message` and `code` properties, and a `to_response()` method that converts the exception to a Python dictionary, which can be returned as a JSON HTTP response.
+The Runway Model SDK defines several custom exception types. All of these exceptions derive from the base ``RunwayError`` class, which itself derives from the standard Python ``Exception`` class. Each ``RunwayException`` contains error `message` and `code` properties, and a `to_response()` method that converts the exception to a Python dictionary, which can be returned as a JSON HTTP response.
```
Comment (agermanidis) [inline docs/source/exceptions.md]:
```suggestion
The Runway Model SDK defines several custom exception types. All of these exceptions derive from the base ``RunwayError`` class, which itself derives from the standard Python ``Exception`` class. Each ``RunwayError`` contains error `message` and `code` properties, and a `to_response()` method that converts the exception to a Python dictionary, which can be returned as a JSON HTTP response.
```

File: docs/source/runway_yaml_file.md [hunk 2]
```diff
@@ -76,4 +75,4 @@ build_steps:
     - `gpu` (boolean, optional, default = `True`): Create a GPU build.
 - `files` (object, optional): A dictionary that defines special behaviors for certain files. All values in this dictionary are specified as paths, with support for the glob character (e.g. `data/*.jpg`).
     - `ignore` (array of strings, optional): A list of file paths to exclude from the build.
-- `build_steps` (array of strings or dictionary values containing the `if_cpu` and `if_gpu` keys, optional): A list of shell commands to run at build time. Use this list to define custom build steps. Each build step is translated to a Docker RUN command, and commands are run in the order they appear in the array.
+- `build_steps` (array of strings or dictionary values containing the `if_cpu` and `if_gpu` keys, optional): A list of shell commands to run at build time. Use this list to define custom build steps. Build steps are run in the order they appear in the array.
```
Comment (agermanidis) [inline docs/source/runway_yaml_file.md]: Maybe a note would be good here about using the `if_cpu` and `if_gpu` keys to only run a given build step on CPU or GPU build respectively?

Comment (brannondorsey) [reply docs/source/runway_yaml_file.md]: Done 😸

**Pull request merged**